Development/src/dataProvider.js

```diff
@@ -212,16 +236,16 @@ const convertDataProviderRequestToHTTP = (type, resource, params) => {
         }
         case GET_MANY_REFERENCE: {
             let total_query;
-            if (params.target !== '' && params[params.source] !== '') {
+            if (params.target !== '' && params.id !== '') {
```

Comment:
Can you talk me through this change, please?

Reply:
The major version bump broke `<ReferenceManyField>` components, params.source was no longer passed to the dataProvider and params.id seemed a fitting replacement.

Reply:
You used to be able to override which of the current resource's properties was the source to be looked up: https://marmelab.com/react-admin/Fields.html#referencemanyfield
Is that not the case any longer?

Reply:
Looks like the react-admin framework maps source to id, so we don't need to do it in the dataProvider any longer.